packages/flutter_test/lib/src/frame_timing_summarizer.dart

```diff
@@ -0,0 +1,152 @@
+
```

Comment:
Nit: remove this new line to pass the license check.

Reply:
Done.